dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -43,7 +43,7 @@ dependent_startup_wait_for=rsyslogd:running
 {# Count how many VLANs require a DHCP relay agent... #}
 {% set num_relays = { 'count': 0 } %}
 {% for vlan_name in VLAN_INTERFACE %}
-{% if VLAN and vlan_name in VLAN and VLAN[vlan_name]['dhcp_servers'] %}
+{% if VLAN and vlan_name in VLAN and VLAN[vlan_name]['dhcp_servers']|length > 0 %}
```

Comment:
>dhcp_servers [](start = 54, length = 12)

Even better, check `'dhcp_servers' in VLAN[vlan_name]`

Reply:
done